spec/emulator_commander_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+
+describe Fastlane::Helper::EmulatorCommander do
+  before do
+    @emulator_commander = Fastlane::Helper::EmulatorCommander
+  end
+
+  describe '#disable_animations' do
+    it 'disables emulator animation' do
+      expect(Fastlane::Actions).to receive(:sh).with("docker exec -i a_name bash -l -c \"adb shell settings put global window_animation_scale 0.0\"")
```

Comment:
By expecting it like you did, you coupled the implementation of `DockerCommander` to to this spec. 
`DockerCommander` can be changed to use something other than `bash` and your test will fail even though everything works. I would expect `DockerCommander` methods to be called.